app/components/icons/face.tsx

```
@@ -0,0 +1,52 @@
+import React from 'react';
+const Face: React.FC<React.SVGAttributes<{}>> = (props) => {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)